src/core/sorted_map.cc

```diff
@@ -181,7 +190,26 @@ unsigned char* ZzlInsert(unsigned char* zl, sds ele, double score) {
   }
 
   /* Push on tail of list when it was not yet inserted. */
-  return zzlInsertAt(zl, NULL, ele, score);
+  return zzlInsertAt(zl, NULL, (unsigned char*)ele.data(), ele.size(), score);
+}
+
+unsigned char* ZzlFind(unsigned char* lp, std::string_view ele, double* score) {
+  unsigned char *eptr, *sptr;
+
+  if ((eptr = lpFirst(lp)) == nullptr)
+    return nullptr;
+  eptr = lpFind(lp, eptr, (unsigned char*)ele.data(), ele.size(), 1);
```

Comment:
can ele be `{null, 0}` ? will it work?

Reply:
Yes, in `lpFind` there are two operations on `ele.data()`, both are checked by the size first, `memcmp` and `lpStringToInt64` both handle nullptr correctly when the size is 0.